src/Action.ts

```diff
@@ -8,6 +8,7 @@ import { ScoredBase, ScoredAction } from './Score'
 export enum ActionTypes {
   TEXT = 'TEXT',
   API_LOCAL = 'API_LOCAL',
```

Comment:
Doesn't matter much but I'd really like to drop the `API_` prefix. `API_LOCAL` should be `CODE` or `CALLBACK` it's just left over from log ago so I wouldn't based the new type on it. 

Are we making this new action specific to Flow forever or eventually it would become general enough to call any `IFTTT` service. Would prefer to make it extensible, and be something like `HTTP_REQUEST` and (which happens to only work for flow now).  Or simply `FLOW`